### PR TITLE
Bootstrapping CT roles. Closes Server#240.

### PR DESCRIPTION
**Pull request by valentinyanakiev**
### Describe the background of your pull request

Bootstrap of CT roles. Currently the user groups are only 'appended' to the user - there is no logic to replace / remove roles (straightforward fix if required).

### Additional context

Read the README to understand how to use it.

### Governance 

- [x] Documentation is added
- [ ] Test cases are added or updated
- [x] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,17 @@ The Data Management panel provides the following capabilities:
 
 Note: the sample data provided here is basic so users are encouraged to look also at the Demo project where additional and more extensive data loading capabilities are available.
 
+## === Bootstrapping Ecoverse roles ===
+
+In order to create Ecoverse profiles, do the following:
+
+1. Set AUTHENTICATION_ENABLED=false environment variable
+1. If you are running an old version of the application (<0.3.*) do a Reset Ecoverse + Sample Data from the Data Management Panel.
+1. Prepare a bootstrap json. Default json file is provided in src/utils/config/authorisation-bootstrap.json. If you'd like to change the name / location of the bootstrap file, set AUTH_BOOTSTRAP_PATH environment variable with the full path to your file.
```
Comment (techsmyth) [inline README.md:54]: Looks like it is searching from the js file, so for me:
Error: Cannot find module 'C:\Users\neil\Documents\Dev_CT\Server\dist\utils\config\authorisation-bootstrap.json'

File: src/domain/user-group/user-group.entity.ts [hunk 2]
```diff
@@ -84,6 +84,8 @@ export class UserGroup extends BaseEntity implements IUserGroup {
 }
 
 export enum RestrictedGroupNames {
-  Admins = 'admins',
+  Admins = 'ecoverse-admins',
```
Comment (techsmyth) [inline src/domain/user-group/user-group.entity.ts]: Should this not be also called "EcoverseAdmins" to be consistent?

File: src/utils/bootstrap/bootstrap.service.ts [hunk 3]
```diff
@@ -25,13 +28,52 @@ export class BootstrapService {
...
   }
 
+  async bootstrapProfiles() {
+    const bootstrapFilePath = this.configService.get<IServiceConfig>('service')
+      ?.authorisationBootstrapPath as string;
+
+    const bootstrapData = await import(bootstrapFilePath);
+    await this.createGroupProfiles(
+      RestrictedGroupNames.Admins,
+      await bootstrapData[RestrictedGroupNames.Admins]
```
Comment (techsmyth) [inline src/utils/bootstrap/bootstrap.service.ts]: this looks strange - awaiting on navigating to data that is already loaded? Probably a good idea to have a defensive check that each expected group is present.

File: src/utils/bootstrap/bootstrap.service.ts [hunk 4]
```diff
@@ -25,13 +28,52 @@ export class BootstrapService {
...
+    );
+  }
+
+  async createGroupProfiles(groupName: string, emails: string[]) {
+    try {
+      emails.forEach(async email => {
+        const userInput = new UserInput();
+        userInput.email = email;
+        userInput.name = 'Imported User';
+        //env variable AUTHENTICATION_ENABLED=false. Otherwise this won't work.
```
Comment (techsmyth) [inline src/utils/bootstrap/bootstrap.service.ts]: Worth putting a check in that accounts are disabled? There is a function for that already via the AccountService.

File: README.md (hunk 1 above)
Comment (valentinyanakiev) [reply README.md:54]: Can't reproduce that. Can you share more details?

File: src/domain/user-group/user-group.entity.ts (hunk 2 above)
Comment (valentinyanakiev) [reply src/domain/user-group/user-group.entity.ts]: Fixed in d6f43ad5ea6b78b4b79d6889962971a5eae009eb

File: src/utils/bootstrap/bootstrap.service.ts (hunk 3 above)
Comment (valentinyanakiev) [reply src/utils/bootstrap/bootstrap.service.ts]: Fixed in 0d3153c183fefa92e83d8ef9c479f3b4f777c3c2

File: src/utils/bootstrap/bootstrap.service.ts (hunk 4 above)
Comment (valentinyanakiev) [reply src/utils/bootstrap/bootstrap.service.ts]: Fixed in d6f43ad5ea6b78b4b79d6889962971a5eae009eb

**Review by techsmyth — APPROVED** [review]

**Pull request merged**